lib/TWCManager/EMS/TED.py

```diff
@@ -51,8 +51,7 @@ def __init__(self, master):
             return None
 
     def debugLog(self, minlevel, message):
-        if self.debugLevel >= minlevel:
-            print("TED: (" + str(minlevel) + ") " + message)
+        self.master.debugLog(minlevel, "TED", message)
```

Comment:
Rather than doing this, we should remove the local debugLog function and reference the central debugLog function instead. I am certainly for the removal of the local debugLog definitions

Reply:
I have to say, that I like this aproach more - instead of always writing the module name as string (possibility of spelling errors = quality) - I always use the local method.

Reply:
I understand what you're saying, but it is a bit hard to reconcile on one hand removing things like precedence operators and spaces that are unnecessary (and harmless) but then adding local functions which are unnecessary as part of the same PR. In my view the debugLog redundancy is worse than unnecessary operators or whitespace, because it requires updating every single file if we added an optional argument to the function later, something which we just did in order to support passing more complex log entries.

Reply:
But updating only at one place per file, not in different places in any file. ;-) To be honest - it's your project - I will change that back.

Reply:
Thanks - whilst I appreciate that you see it as a minor point of contention, it's more that the modular architecture makes it imperative that each module doesn't implement pieces of functionality from other modules. The moment we do that, we lose the dynamic nature of that modular design and lock ourselves in to the local prototypes - and leave ourselves open to inconsistent implementations. I feel that defining the functions in the modules would be a regression as we previously removed most of the local debug functions, putting them back wouldn't make sense to me. I do appreciate your effort to update the PR, thanks.